src/main/java/br/com/sbs/cubatech/category/CategoryController.java

```diff
@@ -73,4 +76,13 @@ public String editCategory(@PathVariable String urlCode, @Valid CategoryForm cat
     public String getAllSubcategories(){
         return "subCategory/viewSubCategoryList";
     }
+
+    @GetMapping("/{urlCode:[a-z-]+}")
+    public String detailsCategory(@PathVariable String urlCode, Model model){
+        Category byUrlCode = categoryRepository.findCategoryActiveByUrlCode(urlCode)
```

Comment:
Esse nome `byUrlCode` não tá muito bom!

```suggestion
        Category category = categoryRepository.findCategoryActiveByUrlCode(urlCode)
```